pliers/extractors/misc.py

```diff
@@ -0,0 +1,78 @@
+'''
+Extractors that operate on Miscellaneous Stims.
+'''
+
+from pliers.stimuli.misc import SeriesStim
+from pliers.extractors.base import Extractor, ExtractorResult
+from pliers.utils import listify
+import scipy
+import numpy as np
+import pandas as pd
+from collections.abc import Iterable
+from importlib import import_module
+import logging
+
+class MetricExtractor(Extractor):
+    ''' Extracts summary metrics from 1D-array using numpy, scipy or custom 
+        functions
+    Args:
+        functions (str, functions or list): function or list of numpy, scipy, 
+            or custom functions to be applied to 1-dimensional numpy arrays.
+            Functions can be passed directly (e.g. passing scipy.stats.entropy) if 
+            the package/module they belong to has been imported, or as strings 
+            (e.g. 'numpy.mean'). Custom functions returning integers or iterables 
+            can also be passed by passing the function itself.
+        var_names (list): optional list of custom alias names for each metric
+        subset_idx (list): subset of Series indices to compute metric on.
```

Comment:
Could this be label based? On Tal's PR I commented that labels are not a requirement for pandas Series, but by default they do get added when using the converter. It seems quite clunky to have to create a boolean index and then pass it to this extractor, rather than labels. 

If we're going down this route, we may even want to allow for regex to filter columns. Although I think in most cases we'l want to restrict columns at the level of the first extractor's output.

Reply:
not sure I understand. The labels are added as index at conversion, right?

Reply:
right, this was just a terminology issue, should be solved by updated docstring.